### PR TITLE
Created a policy on the install of the web terminal operator

### PR DESCRIPTION
**Pull request by jforce**
Added a web terminal policy per the guidelines here: [Contributing and deploying community policies with Red Hat Advanced Cluster Management and GitOps](https://cloud.redhat.com/blog/contributing-and-deploying-community-policies-with-red-hat-advanced-cluster-management-and-gitops). Let me know if anything needs changing. 

Signed-off-by: James Force [jforce@redhat.com](mailto:jforce@redhat.com)

### CODE REVIEW

File: community/CM-Configuration-Management/policy-web-terminal-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                  namespace: openshift-operators
+                spec:
+                  name: web-terminal
+                  channel: fast
+                  installPlanApproval: Automatic
+                  source: redhat-operators
+                  sourceNamespace: openshift-marketplace
+          remediationAction: inform
+          severity: medium
+  remediationAction: enforce
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-web-terminal-operator.yaml]: Can you change the value here from `enforce` to `inform` just to match our best practice of not enforcing policies out of the box?

File: community/README.md [hunk 2]
```diff
@@ -137,6 +137,7 @@ Policy  | Description | Prerequisites
...
 [Policy to install and configure the ODF LVM Operator](./CM-Configuration-Management/policy-lvm-operator.yaml) | Use this policy to install and configure the Openshift Data Foundation LVM Operator | Requires OpenShift 4.10 or later. Nodes must have additional disks that will be used to provide storage.
 [Policy to configure a `ManagedClusterSetBinding`](./CM-Configuration-Management/policy-managedclustersetbinding.yaml) | Use this policy to create a namespace named `policies` and bind it to the `ClusterSet` named `default`. This allows policies created in that namespace to use the cluster placement with any managed clusters in the default cluster set. | This policy should only be placed on the hub cluster.
+[Policy to install the Red Hat Web Terminal Operator](./CM-Configuration-Management/policy-web-terminal-operator.yaml) | OpenShift 4.10 is required. See [About the web terminal in the web console](https://docs.openshift.com/container-platform/4.10/web_console/odc-about-web-terminal.html) for more details.
```
Comment (gparvin) [inline community/README.md]: Guessing 4.10 or newer would work?

**Review by gparvin — COMMENTED** [review]
Thanks for the contribution, I added a couple minor comments.

File: community/README.md (hunk 2 above)
Comment (jforce) [reply community/README.md]: Correct, now fixed

File: community/CM-Configuration-Management/policy-web-terminal-operator.yaml (hunk 1 above)
Comment (jforce) [reply community/CM-Configuration-Management/policy-web-terminal-operator.yaml]: Sure no problem, done

**Comment by jforce** [comment]
@gparvin Thanks for the feedback, I've resolved the issues

**Review by gparvin — APPROVED** [review]

**Comment by gparvin** [comment]
Thanks for this contribution!

**Pull request merged**